comparisons/comparison-cadence.md

```diff
@@ -200,8 +200,8 @@ states:
       refName: processfilefunction
       arguments:
         filename: "${ .file.name }"
-    actionDataFilter:
-      dataResultsPath: "${ .processed }"
+    actionFilter:
+      results: "${ .processed }"
```

Comment:
I think naming them the same thing is more confusing honestly. As mentioned in pr description this uses natural language concept which imo is better:
"A state data filter can filter the state **input** and then filter the state **output**"
"A state event filter can filter the event **data** and merge it **toStateData **element X"
"An event filter can filter elements **fromStateData**, it can also filter the action **results** and merge them **toStateData** element Y"

Reply:
It's a nit; I'm fine if we disagree on this so I'll only add that for actions could be considered to have 'output'.

Also, fwiw, that description appears to be referring to the wrong filter in the 3rd line.  It appears to be actionDataFilter but says 'An event filter'.

Reply:
oops ok will fix. sorry. 
so actions should have "results" / "result" or "output. I am fine with any of those. Just let me know what makes most sense

Reply:
If nobody else weighs in, let's just stick with results I guess.